pkg/image/generated/internalclientset/typed/image/internalversion/imagestream.go

```diff
@@ -154,3 +156,17 @@ func (c *imageStreams) Patch(name string, pt types.PatchType, data []byte, subre
 		Into(result)
 	return
 }
+
+// Secrets takes image.ImageStream name, label and field selectors, and returns the list of Secrets that match those selectors.
+func (c *imageStreams) Secrets(imageStreamName string, opts v1.ListOptions) (result *api.SecretList, err error) {
+	result = &api.SecretList{}
+	err = c.client.Get().
+		Namespace(c.ns).
+		Resource("imagestreams").
+		Name(imageStreamName).
+		SubResource("secrets").
+		VersionedParams(&opts, scheme.ParameterCodec).
```

Comment:
@sttts @deads2k we don't have this in legacy client, not sure if it is needed or just missing in legacy.